src/iris/api.py

```diff
@@ -831,6 +831,41 @@ def gen_where_filter_clause(connection, filters, filter_types, kwargs):
     return where
 
 
+def generate_grouped_query(query, columns, fields):
```

Comment:
Columns and fields are kind of synonyms which make the readability here difficult. 

Could we rename one of them to "allowed_fields" or similar to more easily reflect their usage as a field allowlist?

Reply:
changed to make it clearer